R/run.R

```diff
@@ -546,6 +551,10 @@ validate_orderly_directory <- function(name, root_path, call) {
     cli::cli_abort(err, call = call)
   }
 
+  find_entrypoint_filename(
```

Comment:
this is just being called for the warning/error? A comment perhaps? Will this be removed after we remove the deprecation warning?

Reply:
yep added a comment, this is just for the deprecation warning